src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const convertStyles = {};
+
+  const stylesString = sourceString.split(';');
```

Comment:
You can improve your code, try don't create additional variables and instead of `forEach` use `reduce`

Reply:
Example:

```
     const cssStyles = sourceString
         .trim()
         .split(';')
         .filter(style => style.trim())
         .reduce((acc, style) => {some code....}
         
     return cssStyles;    
```

Reply:
Hello Volodymyr, added changes.